beacon-chain/utils/flags.go

```diff
@@ -26,4 +26,10 @@ var (
 		Name:  "pubkey",
 		Usage: "Validator's public key. Beacon chain node will listen to VRC log to determine when registration has completed based on this public key address.",
 	}
+	// BeaconRPCPort defines a beacon node RPC port to open.
+	BeaconRPCPort = cli.StringFlag{
+		Name:  "beacon-rpc-port",
```

Comment:
Let's think the name here, do we really need beacon? If a beacon code only opens up one kind of rpc port, it should just be `rpc-port` right?

Reply:
Good point, will change